src/spu/spu.cc

```diff
@@ -805,6 +827,8 @@ void PCSX::SPU::impl::playADPCMchannel(xa_decode_t *xap) {
 
 long PCSX::SPU::impl::init(void) {
     spuMemC = (uint8_t *)spuMem;  // just small setup
+    captureBuffer = new CaptureBuffer();
```

Comment:
We probably want to delete this one around shutdown :)

That being said, why not simply making it part of the class itself instead of allocating it?

Reply:
Not a fan of putting all these arrays on the stack. But sure :)

Reply:
Oh but that's not the stack, right? The SPU object is created as a unique_ptr from the main emulator code. It's going to be heap allocated.

Reply:
Ahh, didn't notice that!